shared/types/index.ts

```diff
@@ -111,6 +115,8 @@ export interface CollectionItem {
   title?: string
   description?: string
   images?: ImageProps[]
+  image_srcset_medium?: string
```

Comment:
Aren't `srcset_default`, `srcset_medium` and `srcset_small` already props of each image in the `images` array? What, then, are `image_srcset_medium` and `image_srcset_small`?

Reply:
I simplified this